flags.css

```diff
@@ -1,5 +1,17 @@
 /* Indonesia */
-
+#indonesia {
+    height: 200px;
+    width: 300px;
+    border: 1px solid black
```

Comment:
be aware that adding a border to an element makes it bigger unless you set the box-sizing to border box